freqtrade/tests/test_configuration.py

```diff
@@ -300,7 +300,8 @@ def test_setup_configuration_with_arguments(mocker, default_conf, caplog) -> Non
         'backtesting',
         '--ticker-interval', '1m',
         '--live',
-        '--realistic-simulation',
+        '--enable-position-stacking',
+        '--disable-max-market-positions',
```

Comment:
The name is probably correct but it's getting to be painfully long, should we have a shorter alias for it also?